### ISSUE TITLE
Errors and failure on the second screen

### ISSUE THREAD

**Issue by Girtti** [fishyboteso/fishyboteso#101]
Issue: laptop and the second screen. If you use it with laptop, everything works fine. At the second screen it drops errors:

C:\Users\__\AppData\Local\Programs\Python\Python37\lib\runpy.py:125: RuntimeWarning: 'fishy.main' found in sys.modules after import of package 'fishy', but prior to execution of 'fishy.main'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
created backup
launching please wait...
created session gXgNH7EBNOSS4q5pj3Vt4A
waiting for screen...
screen ready, continuing...
Exception in thread Thread-12:
Traceback (most recent call last):
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\threading.py", line 917, in _bootstrap_inner
    self.run()
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\site-packages\fishy\engine\semifisher\engine.py", line 47, in run
    log_raise("FishyQR not found, try to drag it around and try again")
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\site-packages\fishy\helper\helper.py", line 248, in log_raise
    raise Exception(msg)
Exception: FishyQR not found, try to drag it around and try again

Exception in thread Thread-16:
Traceback (most recent call last):
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\threading.py", line 917, in _bootstrap_inner
    self.run()
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\site-packages\fishy\engine\semifisher\engine.py", line 45, in run
    self.window.crop = get_qr_location(self.window.get_capture())
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\site-packages\fishy\engine\common\qr_detection.py", line 25, in get_qr_location
    gray = cv2.bilateralFilter(og_img, 11, 17, 17)
cv2.error: OpenCV(4.5.2) C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-t9hleyt8\opencv\modules\imgproc\src\bilateral_filter.dispatch.cpp:409: error: (-215:Assertion failed) !_src.empty() in function 'cv::bilateralFilter'

Exception in thread Thread-20:
Traceback (most recent call last):
  File "C:\Users\___\AppData\Local\Programs\Python\Python37\lib\threading.py", line 917, in _bootstrap_inner
    self.run()
  File "C:\Users\___\AppData\Local\Programs\Python\Python37\lib\threading.py", line 865, in run
    self._target(self._args, **self._kwargs)
  File "C:\Users\__\AppData\Local\Programs\Python\Python37\lib\site-packages\fishy\engine\semifisher\engine.py", line 45, in run
    self.window.crop = get_qr_location(self.window.get_capture())
  File "C:\Users\___\AppData\Local\Programs\Python\Python37\lib\site-packages\fishy\engine\common\qr_detection.py", line 25, in get_qr_location
    gray = cv2.bilateralFilter(og_img, 11, 17, 17)
cv2.error: OpenCV(4.5.2) C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-t9hleyt8\opencv\modules\imgproc\src\bilateral_filter.dispatch.cpp:409: error: (-215:Assertion failed) !_src.empty() in function 'cv::bilateralFilter'

Exception in thread Thread-23:
Traceback (most recent call last):
  File "C:\Users\___\AppData\Local\Programs\Python\Python37\lib\threading.py", line 917, in _bootstrap_inner
    self.run()
  File "C:\Users\___\AppData\Local\Programs\Python\Python37\lib\threading.py", line 865, in run
    self._target(self._args, **self._kwargs)
  File "C:\Users\___\AppData\Local\Programs\Python\Python37\lib\site-packages\fishy\engine\semifisher\engine.py", line 45, in run
    self.window.crop = get_qr_location(self.window.get_capture())
  File "C:\Users\___\AppData\Local\Programs\Python\Python37\lib\site-packages\fishy\engine\common\qr_detection.py", line 25, in get_qr_location
    gray = cv2.bilateralFilter(og_img, 11, 17, 17)
cv2.error: OpenCV(4.5.2) C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-t9hleyt8\opencv\modules\imgproc\src\bilateral_filter.dispatch.cpp:409: error: (-215:Assertion failed) !_src.empty() in function 'cv::bilateralFilter'

Either it cannot see the fishing spot, or it starts fishing and immediately drops it 
Also, dragging the FishyQR for the bigger size resize it wrong on the 1920*1280

**Comment by adsau59** [comment]
hi https://github.com/Girtti thanks for creating the issue,

could you please install the branch (https://github.com/fishyboteso/fishyboteso/pull/102) and check if the bot is working properly

to install the branch use the following commands
```
pip uninstall fishy
pip install https://github.com/fishyboteso/fishyboteso.git@feature/multi-monitor
```

**Comment by adsau59** [comment]
fixed and pushed in v0.5.6
